src/test/ui/coherence/coherence-projection-ok-orphan.rs

```diff
@@ -1,11 +1,10 @@
-// compile-pass
-// skip-codegen
+// Here we do not get a coherence conflict because `Baz: Iterator`
+// does not hold and (due to the orphan rules), we can rely on that.
+
+// check-pass
```

Comment:
One thing I'm noticing is that we cannot seem to make up our minds re. whether the test description comment should come first or whether the compiletest annotations should...  😂 We should pick one style, stick to it and document it in the rustc guide.

Reply:
>We should pick one style

I picked one style long time ago :smile: 
Now I apply it to all tests I write or edit.

```rust
// Comment applying to the whole test.

// test-kind
// other-headers

#![feature(something)]
#![deny(something)]

item Item1 { ... }

...

item ItemN { ... }

fn main() {}
```

Reply:
Yeah I'm good with making that the one true style :P just need to get everyone else on board also 😄